overlord/devicestate/devicemgr.go

```diff
@@ -274,6 +280,17 @@ func (m *DeviceManager) StartUp() error {
 
 	// TODO: setup proper timings measurements for this
 
+	if m.preseed {
+		// cache system label for preseeding if core20
```

Comment:
can't we do this earlier already in https://github.com/snapcore/snapd/pull/11205/files#diff-3dedbc3a1986cf905aca293ca6d16560d297253859ecaa8bcea2b7dab7143b3cR143 ?